### PR TITLE
PERS-695: Prop para chamar o simulation

### PR DESCRIPTION
**Pull request by LucasTornai**
#### What problem is this solving?

We still need to be able to call simulation for price in products in some cases. This PR makes it so the compatibility layer calls the simulation for the products and update the prices for the simulationBehavior `default`

#### How should this be manually tested?

The following workspace is linked, if you force the query with simulationBehavior = 'default', the product will return with a different price.

https://tornai--storecomponents.myvtex.com/basic?_q=basic&map=ft

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [X] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Screenshots or example usage

simulationBehavior = 'skip' (or pre-PR)
Screen Shot 2020-07-22 at 17 04 33

simulationBehavior = 'default'
Screen Shot 2020-07-22 at 17 06 52

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes

~~I'm still checking whether calling the simulation for regular stores is too slow, but it probably won't interfere in this PR~~
It's fast.
image

### CODE REVIEW

File: node/commons/compatibility-layer.ts [hunk 1]
```diff
@@ -1,13 +1,25 @@
 import { distinct } from '../utils/object'
 import unescape from 'unescape'
+import { Checkout } from '../clients/checkout'
+import R from 'ramda'
```
Comment (thalytafabrine) [inline node/commons/compatibility-layer.ts]: `ramda` is very heavy, so we always choose to try to use native functions or, if it is not possible, import only the necessary functions. can we do it here?

Comment (LucasTornai) [reply node/commons/compatibility-layer.ts]: I made a new commit so I only import the 4 functions I use from ramda.

File: node/commons/compatibility-layer.ts [hunk 2]
```diff
@@ -76,6 +88,28 @@ export const convertBiggyProduct = (
     skus: skus.find(sku => sku.sellers && sku.sellers.length > 0),
   }
 
+  if (simulationBehavior === 'default') {
+    // eslint-disable-next-line no-console
+    console.log("Usando simulation")
```
Comment (thalytafabrine) [inline node/commons/compatibility-layer.ts]: is this log necessary?

Comment (LucasTornai) [reply node/commons/compatibility-layer.ts]: Oops 🙈

**Review by thalytafabrine — APPROVED** [review]

**Pull request merged**